config/CMS.yml

```diff
@@ -2,8 +2,8 @@
 api:
   versions:
     STU3:
-      url: https://dev.bluebutton.cms.fhirservice.net/protected/bluebutton/fhir/v1/
-      patient: 3979
+      url: https://sandbox.bluebutton.cms.gov/v1/fhir/
+      patient: 20140000008325
```

Comment:
If your access token response is now returning a `patient`, we can get rid of this `patient` line here!

Reply:
@jmandel yes it is returning the patient.  It is also accessible via the `userinfo` endpoint.